pinot-servicemanager/docker-bin/start-servicemanager

```diff
@@ -11,23 +11,10 @@ ZK_ADDRESS=${ZK_ADDRESS:-zookeeper:2181}
 
 # Apply one-time deferred configuration that relies on ENV variables
 #
+sed -i "s/\(controller.zk.str\)=.*/\1=${ZK_ADDRESS}/g" etc/pinot-controller.conf
```

Comment:
rather than appending, this just replaces whatever config line had the ZK string with a value from ENV